packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -311,6 +306,24 @@ class MessageList extends Widget<Properties> {
     return result;
   }
 
+  _updateAria(): void {
+    const aria = {
+      role: 'log',
+      atomic: 'false',
+      label: 'Message list',
```

Comment:
Should we provide opportunity to localize these attributes?

Reply:
That's a separate task, I've already accomplished it in a separate PR.